pfcpiface/main.go

```diff
@@ -115,8 +142,13 @@ func main() {
 	ParseJSON(configPath, &conf)
 	log.Println(conf)
 
+	p4rtcSt := &p4rtc{}
 	bessSt := &bess{}
-	intf = bessSt
+	if conf.EnableP4rt {
+		intf = p4rtcSt
+	} else {
+		intf = bessSt
+	}
```

Comment:
remove the declarations outside of if else block and do this? would that work?
```suggestion
	if conf.EnableP4rt {
		intf = &p4rtc{}
	} else {
		intf = &bess{}
	}
```

Reply:
done